db/migrate/20190402185133_add_b_ill_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddBIllToItems < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :items, :bills, foreign_key: true	  
```

Comment:
This migration looks good!

Can you remove the other two migrations that aren't needed before this gets merged?

Reply:
I know that'll break my local branch but I'm not sure what it'll do to all the others.  I can make a new branch and PR so it's cleaner.

Reply:
Nah, that's alright. We can stick with this. It looks like your migrations do what's needed to get where you want to go, so that's fine. 

Thanks for your work on this!